helm-chart/splunk-kubernetes-metrics/templates/configMap.yaml

```diff
@@ -9,14 +9,33 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   fluent.conf: |
-    # system wide configurations
+    # system wide configurations	
     <system>
       log_level {{ or .Values.logLevel .Values.global.logLevel | default "info" }}
     </system>
     <source>
       @type kubernetes_metrics
       tag kube.*
       node_name "#{ENV['SPLUNK_HEC_HOST']}"
+      {{- with $.Values.kubernetes.url }}
+      kubernetes_url {{ . }}
```

Comment:
Redundant parameter which is not used anywhere, remove this.

Reply:
done